docs/ref/contrib/flatpages.txt

```diff
@@ -164,6 +164,13 @@ For more on middleware, read the :doc:`middleware docs
 How to add, change and delete flatpages
 =======================================
 
+.. warning::
+
+    Permissions to add or edit flatpages should be limited to trusted users
+    only. Flatpages are defined by raw HTML and are **not sanitized** by
+    Django. As a consequence, a malicious flatpage can cause various security
+    vulnerabilities, including permission escalation.
```

Comment:
I would suggest slightly different wording:

```suggestion
    Permissions to add or edit flatpages should be restricted to trusted users.
    Flatpages are defined by raw HTML and are **not sanitized** by Django. As a
    consequence, a malicious flatpage can lead to various security
    vulnerabilities, including unauthorized permission escalation.
```

But looks good in general!

Reply:
I agree with your edits apart from adding "unauthorized" permission escalation, since "permission escalation" is a term for an avenue of attack and so always unauthorized.